gcd/src/main/java/org/spine3/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -55,6 +68,7 @@
     private static final String SNAPSHOT = "SNAPSHOT";
```

Comment:
That's how we mark the `AggregateStorageRecord` to be not an event but an aggregate __snapshot__. This string serves as a prefix for all the snapshot IDs in the storage.